src/neo/SmartContract/ApplicationEngine.cs

```diff
@@ -86,7 +86,7 @@ public partial class ApplicationEngine : ExecutionEngine
         /// GAS spent to execute.
         /// </summary>
         public long GasConsumed { get; private set; } = 0;
-        public long GasRefueled { get; private set; } = 0;
+        public long GasRefueledSummary { get; private set; } = 0;
```

Comment:
I mean //</summary>

Reply:
I see.  This is closed. Let's discuss in 2561.